src/commands/publish.ts

```diff
@@ -38,22 +37,21 @@ const publish = async (
 
     createBlockRequest({ displayName, publishedName }, blockData, category)
         .then((res: AxiosResponse) => {
-            const sortedVersions = sortVersions(res.data.versions);
-            const currentVersion = sortedVersions[0].version;
+            const activeVersion = 1;
```

Comment:
Why a fixed version?

Reply:
Publish will always create the first version right? If `publish` is the command to also create a new version, I can fix that when I add that functionality.

Reply:
Not really. How do you publish version 2? You do: `element new version`, and then, `element publish`. That one is expected to publish version 2.

Reply:
Aw good point, I will address that.